addons/sale/views/res_partner_views.xml

```diff
@@ -43,7 +44,7 @@
         <field name="priority" eval="3"/>
         <field name="arch" type="xml">
             <div name="button_box" position="inside">
-                <button class="oe_stat_button" type="object" name="action_view_sale_order"
+                <button class="oe_stat_button" type="action" name="%(act_res_partner_2_sale_order)d"
```

Comment:
```suggestion
                <button
                    class="oe_stat_button"
                    type="action"
                    name=sale.act_res_partner_2_sale_order"
```
As far as I can see, the `%d` annotation is not necessary anymore, we can reference the xml_id instead (dunno which one is the best performance-wise though).